spec/controllers/allbookingscontroller_spec.rb

```diff
@@ -0,0 +1,5 @@
+# spec/controllers/home_controller_spec.rb
+require 'rails_helper'
+require 'simplecov'
```

Comment:
no need to use require "simple cov" here please add in spec_helper file  and spec_helper already required in rails_helper and you are already using "require 'rails_helper'"  so without using "require 'simplecov'" you can get its functionality

Reply:
okay , i will remove.